tasks/chkrootkit.yml

```diff
@@ -1,5 +1,13 @@
+- name: Generate list of enabled chkrootkit tests
+  shell: chkrootkit -l 2>&1 | cut -d ' ' -f 3- | sed -r "s/\b({{CHKROOTKIT_DISABLED_TESTS | join('|')}})\b//g"
+  register: enabled_tests_output
+
+- name: Set variable with enabled chkrootkit tests
+  set_fact:
+    enabled_tests: "{{enabled_tests_output.stdout}}"
```

Comment:
You could also use `{{ enabled_tests_output.stdout }}` in the template and remove the `set_fact` here – up to you whether you want to change it.